src/guide/components/registration.md

```diff
@@ -136,6 +136,6 @@ A lo largo de la guía, utilizamos nombres PascalCase al registrar componentes.
 
 2. `<PascalCase />` hace más obvio que se trata de un componente Vue en lugar de un elemento HTML nativo en las plantillas. También diferencia los componentes Vue de los elementos personalizados (componentes web).
 
-Este es el estilo recomendado cuando se trabaja con plantillas SFC o de cadena. Sin embargo, como se discute en [Análisis de Advertencias de la Plantilla del DOM](/guide/essentials/component-basics#dom-template-parsing-caveats), las etiquetas PascalCase no son utilizables en las plantillas del DOM.
+Este es el estilo recomendado cuando se trabaja con plantillas SFC o de cadena. Sin embargo, como se discute en [Análisis de advertencias de plantilla en el DOM](/guide/essentials/component-basics#in-dom-template-parsing-caveats), las etiquetas PascalCase no son utilizables en las plantillas del DOM.
```

Comment:
En `src/api/built-in-special-attributes.md` linea 95, se escribe como "Advertencias sobre el Procesamiento de las Plantillas en el DOM", pienso que sería mejor este título for consistency sake y para concordar con el titulo del anchor.